charts/generic/pod-http-latency/engine.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: litmuschaos.io/v1alpha1
+kind: ChaosEngine
+metadata:
+  name: nginx-http-chaos
+  namespace: default
+spec: 
+  # It can be active/stop
+  engineState: 'active'
+  appinfo: 
+    appns: 'default'
+    # FYI, To see app label, apply kubectl get pods --show-labels
+    applabel: 'app=nginx'
+    appkind: 'deployment'
+  chaosServiceAccount: pod-http-latency-sa
+  experiments:
+    - name: pod-http-latency
+      spec:
+        components:
+          env:
+            - name: TOTAL_CHAOS_DURATION
+              value: '60' # in seconds
+            
+            - name: LATENCY
+              value: '2000' #in ms
+
+            # port of the target service
+            - name: TARGET_PORT
+              value: "80"
+
+            # port on which the proxy will listen
+            - name: LISTEN_PORT
+              value: "20000"
```

Comment:
Since users will not commonly configure this, we can remove it from the engine and only keep it in the experiment manifest.